sphinx_needs/builder.py

```diff
@@ -92,3 +92,59 @@ def build_needs_json(app: Sphinx, _exception: Exception) -> None:
     needs_builder = NeedsBuilder(app)
     needs_builder.set_environment(env)
     needs_builder.finish()
+
+
+class NeedumlsBuilder(Builder):
+    name = "needumls"
+
+    def write_doc(self, docname: str, doctree: nodes.document) -> None:
+        pass
+
+    def finish(self) -> None:
+        env = unwrap(self.env)
+        needumls = env.needs_all_needumls.values()
+
+        for needuml in needumls:
+            if needuml["save"]:
+                puml_content = needuml["content_calculated"]
+                # check if given save path dir exists
+                save_path = os.path.join(self.outdir, needuml["save"])
+                save_dir = os.path.dirname(save_path)
+                if not os.path.exists(save_dir):
+                    os.makedirs(save_dir, exist_ok=True)
+
+                log.info(f"Storing needuml data: {needuml} to file {save_path}.")
```

Comment:
How does this look like?
Isn't ``{needuml}`` an object? How long can the string be?
I think writing the file path is enough information